Kudu.Services.Web/Tracing/TraceModule.cs

```diff
@@ -150,7 +151,8 @@ private static void OnBeginRequest(object sender, EventArgs e)
 
         public static bool IsRbacWhiteListPaths(string path)
         {
-            return !DisallowedPaths.Any(path.Contains) && _rbacWhiteListPaths.Any(r => r.IsMatch(path));
+            path = path.ToLower();
+            return !DisallowedPaths.Any(p => path.ToLower().Contains(p.ToLower())) && _rbacWhiteListPaths.Any(r => r.IsMatch(path));
```

Comment:
path has already been normalized in the line above, so you can simplify to `!DisallowedPaths.Any(p => path.Contains(p.ToLower()))`

Reply:
Fixed thanks.